recovery.py

```
@@ -0,0 +1,566 @@
+                saved = None
+                for k, v in saved_state.items():
+                    if v.get('symbol') == symbol and v.get('side') == side:
+                        saved = {**v, 'key': k}
+                        break
+                
+                if saved:
+                    stop = saved.get('stop', entry_price)
+                    print(f"  📁 SL desde estado.json: {stop:.8g}")
+                else:
+                    # Buscar en TXT señales
+                    txt_signals = _load_txt_signals()
```

Ruff (0.15.12)

[error] Undefined name `_load_txt_signals`

(F821)